src/script/auth/component/ClientList.tsx

```diff
@@ -91,7 +92,7 @@ const ClientList = ({
       {permanentClients.map(client => (
         <ClientItem
           client={client}
-          clientError={isSelectedClient(client.id) && clientError}
+          clientError={isSelectedClient(client.id) ? clientError : undefined}
```

Comment:
Please do smth like that instead of passing undefined :) 

```suggestion
          {...(isSelectedClient(client.id) && { clientError })}
```

Reply:
its less readable for me but does the same thing... 🤔

Reply:
![image](https://user-images.githubusercontent.com/13432884/191485509-00c2bb4d-10e6-49ee-9a82-ee7e7149d3ce.png)
This is a difference :)

Reply:
<img width="369" alt="Screenshot 2022-09-21 at 12 54 43" src="https://user-images.githubusercontent.com/45733298/191486776-c6a21f84-2ad9-4e03-912f-204bb81d22aa.png">

Reply:
but the result is the same, we're passing it as props

Reply:
Yeah sure, I prefer spread it instead of passing undefined :)